src/main/resources/export/java/Subsystem-imports.java

```diff
@@ -0,0 +1 @@
+${helper.getImports($subsystem, "RobotMap")}
```

Comment:
I assume no EOL at EOF is intentional here?  I know the way blocks work in the output may result in double-spacing...

Reply:
correct